frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit } from '@angular/core';
+import { Offer } from 'app/homepage-offer/offers.model';
+import { AuthService } from 'app/auth.service';
+import { User } from 'app/user';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import { OffersService } from 'app/homepage-offer/offers.service';
+import { ActivatedRoute, Params } from '@angular/router';
+import { environment } from 'environments/environment';
+
+@Component({
+  selector: 'volontulo-create-offer',
+  templateUrl: './create-offer.component.html',
+  styleUrls: ['./create-offer.component.scss']
+})
+export class CreateOfferComponent implements OnInit {
+  createOrEdit = 'Tworzenie';
```

Comment:
I believe that in terms of class fields we should explicitly state access modifiers. I would mark all those below as public e.g. public djangoRoot, public edit etc. @magul Your opinion?